examples/with-claims/api/claims/RolesClaim.ts

```diff
@@ -0,0 +1,91 @@
+import Session, { SessionClaim, SessionClaimValidator, SessionContainer } from "supertokens-node/recipe/session";
+import { roleDB } from "../mockRoleDb";
+
+class RolesClaimClass extends SessionClaim<string[]> {
```

Comment:
Can't we just use extends PrimitiveClaim?

Reply:
We could, but it'd be slightly wrong: this contains an array, not a primitive value.